server/config.go

```diff
@@ -415,11 +418,28 @@ func (c *ScheduleConfig) clone() *ScheduleConfig {
 	}
 }
 
+const (
+	defaultMaxReplicas          = 3
+	defaultMaxSnapshotCount     = 3
+	defaultMaxPendingPeerCount  = 16
+	defaultMaxMergeRegionSize   = 0
+	defaultSplitMergeInterval   = 30 * time.Minute
```

Comment:
Can we default to 1 hour?
I think 1 hour should be enough for TiDB Lighting in most cases.

Reply:
okay